convert2rhel/actions/system_checks/convert2rhel_latest.py

```diff
@@ -31,22 +31,19 @@
 
 logger = logging.getLogger(__name__)
 
-# The SSL certificate of the https://cdn.redhat.com/ server
-SSL_CERT_PATH = os.path.join(utils.DATA_DIR, "redhat-uep.pem")
-CDN_URL = "https://cdn.redhat.com/content/public/convert2rhel/$releasever/$basearch/os/"
+CDN_URL_7 = "https://cdn-public.redhat.com/content/public/addon/dist/convert2rhel/server/7/7Server/x86_64/os/"
+CDN_URL_8 = "https://cdn-public.redhat.com/content/public/addon/dist/convert2rhel8/8/x86_64/os/"
```

Comment:
Can we change to dict with numbers for the keys? That way we don't have to do check for system and grab specific version but can rather loop through the dict and take the first match.

```suggestion
CDN_URLS = {
  7: "https://cdn-public.redhat.com/content/public/addon/dist/convert2rhel/server/7/7Server/x86_64/os/",
  8: "https://cdn-public.redhat.com/content/public/addon/dist/convert2rhel8/8/x86_64/os/"
}
```

Then do a switch on `system_info.version.major`